addons/anonymization/anonymization.py

```diff
@@ -523,7 +523,7 @@ def reverse_anonymize_database(self, cr, uid, ids, context=None):
             data = pickle.loads(base64.decodestring(wizard.file_import))
 
             migration_fix_obj = self.pool.get('ir.model.fields.anonymization.migration.fix')
-            fix_ids = migration_fix_obj.search(cr, uid, [('target_version', '=', '8.0')])
+            fix_ids = migration_fix_obj.search(cr, uid, [('target_version', '=', '9.0')])
```

Comment:
can't you use the version from [release.py](https://github.com/odoo/odoo/blob/915e41b19aaeb92028118a43546b1e3dd8827c1e/openerp/release.py#L15) file instead of updating this file at each version, cf 10a1d443891001b1ba2094600e8635b59901f33d